zuul.d/projects.yaml

```diff
@@ -14,6 +14,7 @@
         - cifmw-end-to-end
         - cifmw-end-to-end-nobuild-tagged
         - cifmw-kuttl
+        - cifmw-base-tcib
```

Comment:
this must also be added to the original template used to generate the file:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci/templates/projects.yaml
else it would be removed whenever we regenerate it (i.e. when we add a new role).

Please keep the alphabetical order :)